packages/core/src/bus/index.ts

```diff
@@ -0,0 +1,11 @@
+export * from './Command';
+export * from './CommandDispatcher';
+export * from './Event';
+export * from './EventDispatcher';
+export * from './EventBus';
+export * from './RetryStartegy';
+export * from './Handler';
+export * from './EventHandler';
+export * from './Exceptions';
+export * from './Decorators';
```

Comment:
Make sure that folder names are consistent

```suggestion
export * from './exceptions';
export * from './decorators';
```